create.py

```diff
@@ -0,0 +1,46 @@
+import os
+from flask import Flask, request, render_template
+from models import *
+from flask import Flask, session, redirect
+from flask_session import Session
+from sqlalchemy import create_engine
+from sqlalchemy.orm import scoped_session, sessionmaker
+
+app = Flask(__name__)
+app.config['SQLALCHEMY_DATABASE_URI'] = os.getenv("DATABASE_URL")
+app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+app.config["SESSION_PERMANENT"] = False
+app.config["SESSION_TYPE"] = "filesystem"
+engine = create_engine(os.getenv("DATABASE_URL"))
+
+Session(app)
+db.init_app(app)
+
+@app.route('/')
+def hello_world():
+    return redirect('/register')
+@app.route('/register', methods = ['POST','GET'])
+def register():
+    db.create_all()
+    if request.method == 'POST':
+        data = request.form
+        userdata = USERS(request.form['email'],request.form['psw'])
+        user = USERS.query.filter_by(emailid=request.form['email']).first()
+        if user is not None:
+            print("User is already existing. Please try to register with a new")
+            var1 = "Error: User is already existing. Please try to register with a new one"
+            return render_template("reg.html", var1 = var1)
+        db.session.add(userdata)
+        db.session.commit()
+        print("Registration Success")
+        var1 ='Registration Success'
+        return render_template("reg.html", var1 = var1)
+    else:
+        return render_template("reg.html")
+    
+
+@app.route('/admin')
+def admin():
+    data = USERS.query.all()
```

Comment:
Users data needs to be printed in the descending order of timestamp.

Reply:
Updated bro please check once so that I can start login part

Reply:
looks good

Reply:
shall i move the  contents of create.py to application.py
I kept it in another because i thought it may reflect to any merge conflicts